### PR TITLE
Update manage-license-requests.md

### PR DESCRIPTION
**Pull request by msbemba**
Added related content (Self Service purchase Faq) to this document.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8769

File: microsoft-365/commerce/licenses/manage-license-requests.md [hunk 1]
```diff
@@ -76,4 +76,5 @@ When you return to the **Requests** list, you see the message **You’re using y
 
 [Assign licenses to users](../../admin/manage/assign-licenses-to-users.md) (article)\
 [Move users to a different subscription](../subscriptions/move-users-different-subscription.md) (article)\
-[Buy or remove subscription licenses](buy-licenses.md) (article)
+[Buy or remove subscription licenses](buy-licenses.md) (article)\
+[Self-service purchase FAQ](https://docs.microsoft.com/en-us/microsoft-365/commerce/subscriptions/self-service-purchase-faq?view=o365-worldwide)
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/licenses/manage-license-requests.md]:
```suggestion
[Self-service purchase FAQ](/microsoft-365/commerce/subscriptions/self-service-purchase-faq)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Review by cmcatee-MSFT — APPROVED** [review]

**Pull request merged**